hack/jenkins/e2e-runner.sh

```diff
@@ -89,13 +89,19 @@ function get_latest_gci_image() {
 function get_latest_docker_release() {
   # Typical Docker release versions are like v1.11.2-rc1, v1.11.2, and etc.
   local -r version_re='.*\"tag_name\":[[:space:]]+\"v([0-9\.r|c-]+)\",.*'
-  local -r latest_release="$(curl -fsSL --retry 3 https://api.github.com/repos/docker/docker/releases/latest)"
-  if [[ "${latest_release}" =~ ${version_re} ]]; then
-    echo "${BASH_REMATCH[1]}"
-  else
-    echo "Malformed Docker API response for latest release: ${latest_release}"
-    exit 1
-  fi
+  local -r releases="$(curl -fsSL --retry 3 https://api.github.com/repos/docker/docker/releases)"
+  # The GitHub API returns releases in descending order of creation time so the
+  # first one is always the latest.
+  # TODO: if we can install `jq` on the Jenkins nodes, we won't have to craft
```

Comment:
The kubekins-test image has jq: https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-image/Dockerfile#L34